worker-workflow/src/main/java/com/github/cafdataprocessing/WorkflowEvaluator.java

```diff
@@ -53,10 +50,11 @@ public static void evaluate(final Document document, final String workflowAsJava
      * @param document Document to set post processing information on.
      * @param workflowStorageReference DataStore reference for a workflow that can be executed in post processing.
      */
-    private static void setWorkflowPostProcessingScript(final Document document, final String workflowStorageReference)
+    private static void setWorkflowPostProcessingScript(final Document document, final String workflowStorageReference) throws ScriptException
     {
         final Script script = document.getTask().getScripts().add();
         script.setScriptByReference(workflowStorageReference);
         script.setName("postProcessingScript");
+        script.load();
```

Comment:
This will result in the script being downloaded from the data store (at least the first time - it would be cached after that).  But since you have the script right there instead of loading this one you could add it as a second script, uninstall it so that it doesn't get passed on, and then load it instead - that way there is no data store download.